src/components/CampaignLocations/CampaignLocationsListItem.tsx

```diff
@@ -63,10 +63,10 @@ export const CampaignLocationsListItem: FunctionComponent<
           ]}
         >
           {hasExpired
-            ? i18n.t("customerQuotaScreen.campaignExpiredOn", {
+            ? i18nt("customerQuotaScreen", "campaignExpiredOn", undefined, {
```

Comment:
Oooh, actually rather than doing this `undefined` thing, what if `i18n` took in two arguments

```
i18n(["customerQuotaScreen", "campaignExpiredOn", ...], {variables})
```

First array can be infinite size, so we can be flexible for all levels of nesting. What do you think?

Reply:
In that case this can be an array of two strings rather than having `undefined`

Reply:
After much discussion and deliberation, we have decided this may not be worth the effort to fix now, since it will take a much more complicated solutions than necessary to both retain the type checking and have a succinct function signature.